pkg/cmd/pulumi/pulumi.go

```diff
@@ -498,7 +498,9 @@ func getUpgradeMessage(latest semver.Version, current semver.Version) string {
 		msg += "run \n   " + cmd + "\nor "
 	}
 
-	msg += "visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes."
+	msg += "visit https://pulumi.com/docs/reference/install/ for manual instructions\n\n"
+	msg += "Release Notes:\n"
+	msg += "https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md"
```

Comment:
this feels suboptimal if we ever rename the branch to be main

Reply:
that was my first thought, but I found this bug
https://github.com/pulumi/pulumi/issues/10752

it reminds me of this silent bug
https://github.com/pulumi/pulumi/issues/10508

I could add an integration test, but then the dependencies seem to get really hard to think about because then our tests depend on the latest flag from the service and the changelog on the last release being correct despite not preventing the last release from being incorrect. :woman_shrugging:

Reply:
I really don’t think you are incorrect adding it / it’s just something we need to be aware of it we rename the default branch is all

Reply:
when are we renaming the default branch?

Reply:
It's not on the roadmap afaik. 

Our policy is that new repos use main and old repos continue to use master.